client/index.html

```diff
@@ -0,0 +1,48 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="http://fonts.cdnfonts.com/css/stickman" rel="stylesheet" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Thoughts in C</title>
+  </head>
+  <body>
+    <div class="titleWrap">
+      <div class="titleDiv">
+        <h1 id="header">Thoughts in C</h1>
+      </div>
+    </div>
+
+    <section>
+      <form id="post">
```

Comment:
Setting some of the elements to be required would be good so you can't submit an empty form